protocol/x/perpetuals/keeper/perpetual.go

```diff
@@ -186,6 +186,33 @@ func (k Keeper) ModifyPerpetual(
 	return perpetual, nil
 }
 
+func (k Keeper) SetPerpetualMarketType(
```

Comment:
nit: Can we change it such that we only set the type if it's not already set to CROSS/ISOLATED? With a comment indicating this should be removed once upgrading markets is supported.

Reply:
Also some simple tests would be preferred for this function.

Reply:
Why not just leave this as is considering there is no msg other than the upgrade that will call into this, rather than adding the restriction and removing it down the line?

Reply:
done